.gitignore

```diff
@@ -6,14 +6,6 @@
 # macOS
 .DS_Store
 
-# qt-creator
```

Comment:
People should set up global ignores, I'm going to clean up this file soon it's impossible to look at as is.

Reply:
some people managed to add their whole build dir and commit that. this was a precaution